postile/templates/index-debug.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8' />
+    <title>Postile/OpenMapTiles example DEBUG mode</title>
+    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.3.1/mapbox-gl.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.3.1/mapbox-gl.css' rel='stylesheet' />
```

Comment:
I kinda wanted you to use openlayers ;-) But it's fine for now, we could always make another example with openlayers later.

Reply:
Don't matter to me, why using OpenLayers instead of mapbox-gl-js ? 
Anyway, feel free to add a switcher to preview with both libraries.

Reply:
For the sake of using a community-oriented project :-) But really, mapbox-gl is fine, and it's also a sort of reference implementation so it makes sense to have it anyway. I'll add a switcher if I find the time.